config/dev.webpack.config.js

```diff
@@ -3,17 +3,13 @@
 const _ = require('lodash');
 const webpackConfig = require('./base.webpack.config');
 const config = require('./webpack.common.js');
-const history = require('connect-history-api-fallback');
-const convert = require('koa-connect');
 
-webpackConfig.serve = {
-    content: config.paths.public,
+webpackConfig.devServer = {
+    contentBase: config.paths.public,
+    hot: true,
     port: 8002,
-    dev: {
-        publicPath: '/insights/platform/advisor'
-    },
-    // https://github.com/webpack-contrib/webpack-serve/blob/master/docs/addons/history-fallback.config.js
-    add: app => app.use(convert(history({})))
```

Comment:
Best I can tell, this was for preserving browser history? or something?  Anyhow appears to work just fine when using `historyApiFallback: true` @ryelo @karelhala either of yah intimately familiar with this?

Reply:
@AllenBW yeah, you are right. This was mainly to preserve browser history and enable refreshing when new changes to code happend.